docs/standard/microservices-architecture/docker-application-development-process/index.md

```diff
@@ -19,16 +19,18 @@ ms.topic: article
 
 Whether you prefer a full and powerful IDE or a lightweight and agile editor, Microsoft has tools that you can use for developing Docker applications.
 
-**Visual Studio with Tools for Docker**. If you are using Visual Studio 2015, you can install the [Visual Studio Tools for Docker](https://marketplace.visualstudio.com/items?itemName=MicrosoftCloudExplorer.VisualStudioToolsforDocker-Preview) add-in. If you are using Visual Studio 2017, tools for Docker are already built-in. In either case, the tools for Docker let you develop, run, and validate your applications directly in the target Docker environment. You can press F5 to run and debug your application (single container or multiple containers) directly into a Docker host, or press CTRL+F5 to edit and refresh your application without having to rebuild the container. This is the simplest and most powerful choice for Windows developers targeting Docker containers for Linux or Windows.
+**Visual Studio (for Windows)**. When developing Docker-based applications with Visual Studio, it is recommended to use Visual Studio 2017 or higher which comes with tools for Docker are already built-in. The tools for Docker let you develop, run, and validate your applications directly in the target Docker environment. You can press F5 to run and debug your application (single container or multiple containers) directly into a Docker host, or press CTRL+F5 to edit and refresh your application without having to rebuild the container. This is the most powerful development choice for Docker based apps.
```

Comment:
When developing Docker-based applications with Visual Studio, it is recommended to use Visual Studio 2017 or higher which comes with tools for Docker are already built-in. ->
To develop Docker-based applications, use Visual Studio 2017 or later versions that comes with tools for Docker already built-in.

Docker based apps -> Docker-based apps

Reply:
Fixed.